test/deriving/test.ml

```diff
@@ -35,8 +35,8 @@ Line _, characters 25-33:
 Error: Deriver foo is needed for bar, you need to add it before in the list
 |}]
 
-type t = int [@@deriving foo, bar]
+type nonrec int = int [@@deriving foo, bar]
```

Comment:
What is that diff about?

Reply:
If memory serves, it was to have the same expectation
across the various supported versions (with the initial
source code, some would say `t` and others would say
`int` when printing back the type).